openlibrary/plugins/openlibrary/home.py

```diff
@@ -238,6 +238,7 @@ def get_authors(doc):
     work = book.works and book.works[0]
     d.authors = get_authors(work if work else book)
     cover = work.get_cover() if work and work.get_cover() else book.get_cover()
+    logger.error("format_book_data(%s) work=%s, d.authors=%s" % (book, work, d.authors))
```

Comment:
Why are we logging an error here?

Reply:
I was looking to see why pytests were failing.  Reverted.